pallets/schemas/src/tests.rs

```diff
@@ -77,3 +77,35 @@ fn register_schema_id_deposits_events_and_increments_schema_id() {
 		assert_ok!(SchemasPallet::register_schema(Origin::signed(sender), serialized_fields1));
 	})
 }
+
+#[test]
+fn get_existing_schema_by_id_should_return_schema() {
+	new_test_ext().execute_with(|| {
+		// arrange
+		let sender: AccountId = 1;
+		let serialized_fields = Vec::from("foo,bar,bazz".as_bytes());
+		assert_ok!(SchemasPallet::register_schema(
+			Origin::signed(sender),
+			serialized_fields.clone()
+		));
+
+		// act
```

Comment:
left over comments, there are a few in the file

Reply:
If you mean `Act` it is related to Arrange/Act/Assert pattern
https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/

Reply:
oh cool did not know. Thanks for sharing.